webapp/views.py

```diff
@@ -393,6 +394,44 @@ def get_notice(notice_id, **kwargs):
             404,
         )
 
+    releases = {
+        release.codename: release.version
+        for release in db_session.query(Release).all()
+    }
+
+    package_descriptions = {}
+    package_versions = {}
+    release_packages = SortedDict()
+
+    if notice.release_packages:
+        for codename, pkgs in notice.release_packages.items():
```

Comment:
Is there anything wrong with using the word `packages`? Is it already used somewhere else in this file?

Some people disagree (like [the whole of the Go community](https://www.reddit.com/r/golang/comments/8wxwgv/why_does_go_encourage_short_variable_names/) :roll_eyes:) but personally I feel strongly that readability, clarity and accessibility of our code for all to use is more important than the time it takes to write a longer variable name.

Unless you disagree, could you try to avoid shortened or abbreviated variable names wherever possible?